packages/accounts-password/password_client.js

```diff
@@ -223,6 +224,10 @@ Accounts.forgotPassword = (options, callback) => {
     return reportError(new Meteor.Error(400, "Must pass options.email"), callback);
   }
 
+  if (!options.detailedErrorFlag){
+    options.detailedErrorFlag = false
```

Comment:
Im not sure if meteor code style enforces the use of ";", but everywhere else is used. It might be better to keep using it in the new code.

Reply:
@pmogollons thank you for pointing it out. I will get right to it.